scene/2d/tile_map.cpp

```diff
@@ -270,8 +270,10 @@ void TileMap::_update_dirty_quadrants() {
 		return;
 	if (!is_inside_tree())
 		return;
```

Comment:
Well, it is set to true by various other functions, which might _potentially_ run before `enter_tree` -- I guess it is better to be save than sorry, especially when it is just one if statement :smiley: